Sources/SIL/SILAnalysis.swift

```diff
@@ -195,11 +196,193 @@ extension Array: AlphaConvertible where Element: AlphaConvertible {
     }
 }
 
+func substitute(_ type: Type, using s: (String) -> Type) -> Type {
```

Comment:
Let's have this as an extension method on `Type` to be consistent with things like `Operator.alphaConverted` etc?

Reply:
Sure, can do.